src/pudl/metadata/resources/ferc1.py

```diff
@@ -839,6 +839,27 @@
         "sources": ["ferc1"],
         "include_in_database": False,
     },
+    "denorm_balance_sheet_assets_ferc1": {
+        "description": "Denormalized table that contains FERC balance sheet asset information.",
+        "schema": {
```

Comment:
Do we want to include `primary_key` definitions in the denormalized table schemas if we think they should exist? They'll get checked for being non-null and unique automatically then, and it would probably be good to know if we expect them to be valid primary keys but they are not.

Reply:
Yeah good catch, checking the primary keys will probably be a good sanity check